rholang/src/main/scala/coop/rchain/rholang/interpreter/normalize.scala

```diff
@@ -981,7 +988,7 @@ object ProcNormalizeMatcher {
             case _: BundleWrite     => Bundle(targetResult.par, writeFlag = true, readFlag = false)
             case _: BundleEquiv     => Bundle(targetResult.par, writeFlag = false, readFlag = false)
           }
-          res <- if (targetResult.par.connectiveUsed) {
+          res <- if (targetResult.par.connectiveUsed && !isSimpleType(targetResult.par.connectives)) {
```

Comment:
It seems like currently this won't work because we don't capture free vars and wildcards when `\/` or `~` is used as a connective.
I'm not sure if I should add fields to `ProcVisitOutputs` that capture free vars and wildcards all the time or if I should change the logic so that free vars and wildcards are captured all the time in `ProcVisitOutputs.knownFree`?

Reply:
I think the root cause of this bug is using different data to check `if` there's a bug (`outputs.par.connectiveUsed`), and different to check `what's` the bug (`outputs.knwonFree.*`). I think what we should do is rely on `knownFree`, just as `Interpreter` does in `normalizeTerm`.

The only question that remains is whether we should at all allow connectives in the top level of a bundle. My examples above do that, but I'm not sure if they should be allowed.

@pyrocto @guardbotmk3 Should we allow `logical connectives` in top level of a bundle? How about `simple type connectives`?